lib/oban/queue/producer.ex

```diff
@@ -201,7 +201,8 @@ defmodule Oban.Queue.Producer do
   end
 
   defp schedule_refresh(%State{} = state) do
-    timer = Process.send_after(self(), :refresh, state.refresh_interval)
+    cooldown = Breaker.jitter(state.refresh_interval, mode: :add, mult: 1)
```

Comment:
The refresh interval needs to use `:dec` to be _at least as frequent as the interval_. Otherwise, with a tight configuration, a plugin like Lifeline could erroneously rescue jobs from a running producer.

```suggestion
    cooldown = Breaker.jitter(state.refresh_interval, mode: :dec, mult: 0.5)
```

Reply:
Oh, I thought the interval was expected to be the minimum time :)